frontend/src/core/user/components/FileDownload.js

```diff
@@ -0,0 +1,56 @@
+/* @flow */
+
+import * as React from 'react';
+import { Localized } from 'fluent-react';
+
+import type { NavigationParams } from 'core/navigation';
+
+
+type Props = {|
+    parameters: NavigationParams,
+|};
+
+
+/*
+ * Render a Sign Out link.
+ */
+export default class SignOut extends React.Component<Props> {
+    downloadTranslations = () => {
+        document.getElementById('download-form').submit();
+    }
+
+    render() {
+        const { parameters } = this.props;
+
+        let csrfToken = '';
+        const rootElt = document.getElementById('root');
+        if (rootElt) {
+            csrfToken = rootElt.dataset.csrfToken;
+        }
+
+        return <React.Fragment>
+            <Localized
+                id="user-UserMenu--download-translations"
+                glyph={
+                    <i className="fa fa-cloud-download-alt fa-fw"></i>
+                }
+            >
+                <button onClick={ this.downloadTranslations }>
+                    { '<glyph></glyph>Download Translations' }
+                </button>
+            </Localized>
+
+            <form
+                id="download-form"
+                action="/download/"
+                className="hidden"
+                method="POST"
```

Comment:
Why does the view `download` require `POST`? It could be so much better if it was a get, then this could simply be a link. Browsers nowadays don't hide the page you where in if the target of the link is a download.